Specs/Scene/ImageryLayerSpec.js

```diff
@@ -156,7 +156,7 @@ defineSuite([
         });
     }
 
-    it('reprojects web mercator images', function() {
+    it('reprojects web mercator images when necessary', function() {
```

Comment:
Are any other new unit tests needed, e.g., request completely inside +/- ~85 latitude and completely outside?